src/Symfony/Component/Yaml/Inline.php

```diff
@@ -33,6 +33,18 @@ class Inline
     private static $objectForMap = false;
     private static $constantSupport = false;
 
+    public static function initialize(int $flags, int $parsedLineNumber = null)
```

Comment:
Is this new method intended?

Reply:
The idea was to not have to accept `null` values in `Inline::parse()`. But I have moved it to its own PR: #24060